python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -3242,86 +3242,63 @@ def test_ndim():
 
 
 @pytest.mark.parametrize(
-    "arr",
-    [
-        np.random.normal(-100, 100, 1000),
-        np.random.randint(-50, 50, 1000),
-        np.zeros(100),
-        np.repeat([-0.6459412758761901], 100),
-        np.repeat(np.nan, 100),
-        np.array([1.123, 2.343, np.nan, 0.0]),
-    ],
-)
-@pytest.mark.parametrize(
-    "decimal",
+    "decimals",
     [
+        -3,
         0,
-        1,
-        2,
-        3,
-        4,
         5,
-        6,
-        7,
-        8,
-        9,
-        10,
-        11,
-        12,
-        13,
-        14,
-        15,
-        16,
-        17,
-        pytest.param(
-            -1,
-            marks=[
-                pytest.mark.xfail(reason="NotImplementedError: decimals < 0")
-            ],
-        ),
+        pd.Series([4, 3, -6], index=["x", "y", "z"]),
+        gd.Series([-2, 1, 12], index=["x", "y", "z"]),
+        {"x": 15, "y": -2, "z": 8},
     ],
 )
-def test_round(arr, decimal):
-    pser = pd.Series(arr)
-    ser = gd.Series(arr)
-    result = ser.round(decimal)
-    expected = pser.round(decimal)
+def test_dataframe_round(decimals):
```

Comment:
Since we defaulted `HALF_EVEN` rounding method, should we create a test (guard) case for the proper behavior? For example:

```
5.5 ~> 6
6.5 ~> 6
7.5 ~> 8
-7.5 ~> -8
...
```

Reply:
NM, already included in test cases.